treeherder/services/pulse/consumers.py

```diff
@@ -162,6 +162,36 @@ def on_message(self, body, message):
         )
         message.ack()
 
+class PulsesConsumer(PulseConsumer):
+    queue_suffix = env("PULSE_SOMETHING", default="need")
```

Comment:
The values should be `PULSE_QUEUE_NAME` and `queue`.

This will require updating the values in `app.json`. Remove the other two and only use this.